src/Aquarius.ONE.ClientSDK/Utilities/RestHelper.cs

```diff
@@ -12,16 +12,19 @@ namespace ONE.Utilities
 {
     public class RestHelper
     {
-        public RestHelper(AuthenticationApi authentication, PlatformEnvironment environment, bool continueOnCapturedContext, bool logRestfulCalls)
+        public RestHelper(AuthenticationApi authentication, PlatformEnvironment environment, bool continueOnCapturedContext, bool logRestfulCalls, bool throwAPIErrors)

```

Comment:
@stevenhaching  - do you know if teams are using this RestHelper class directly?  If so, maybe make this bool optional here.

Reply:
You are totally right.  I will add this

Reply:
Done